src/Components/Nav/FloatingBtn.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styled from "styled-components";
+
+function FloatingBtn(props) {
+  return (
+    <Wrapper onClick={props.scrollToTop}>
+      <Box />
+    </Wrapper>
+  );
+}
+
+export default FloatingBtn;
+
+const Wrapper = styled.div`
+  display: block;
```

Comment:
```suggestion
const Wrapper = styled.div`
```
div 태그인데 block? 딱밤 때리러 갑니다 🤛

Reply:
수정 완료

Reply:
때리지는 마요... @saengmotmi